library/src/main/java/com/chuckerteam/chucker/internal/ui/HomePageAdapter.kt

```diff
@@ -0,0 +1,36 @@
+package com.chuckerteam.chucker.internal.ui
+
+import android.content.Context
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentStatePagerAdapter
+import com.chuckerteam.chucker.R
+import com.chuckerteam.chucker.internal.ui.error.ErrorListFragment
+import com.chuckerteam.chucker.internal.ui.transaction.TransactionListFragment
+import java.lang.ref.WeakReference
+
+/**
+ * @author Olivier Perez
+ */
```

Comment:
You can remove this comment while you are at it, we are not really adding authorship like that anymore.